blueprints/in-repo-engine/routable-files/lib/__name__/addon/routes.js

```diff
@@ -0,0 +1,3 @@
+export default function() {
```

Comment:
I don't want to block this from being merged as I believe it will help many people add engines to their apps, but I'd like it if we could fix https://github.com/dgeb/ember-engines/issues/64 before releasing a version with this blueprint.